### DISCUSSION TITLE
Voice keyboard using Whisper on mobile

### DISCUSSION THREAD

**Discussion by lrq3000** [openai/whisper#1784] [General]
I saw someone ask about this so here are the mobile voice keyboards i found using Whisper on Android (others maybe can share for iOS):

* the best one imho, supporting larger models and multilingual, but limited to 30s inputs: FUTO Voice Input https://play.google.com/store/apps/details?id=org.futo.voiceinput sourcecode: https://gitlab.futo.org/alex/voiceinput
* less good because uses a smaller model and only supports english but can process infinitely long streams of audio: https://play.google.com/store/apps/details?id=kaizo.co.WhisperVoiceKeyboard likely sourcecode but not sure this is the same project: https://github.com/MichaelMcCulloch/WhisperVoiceKeyboard

**Comment by vickylahkarbytes** [comment]
Thanks for the info. Please do update here if you have come across any new updates.

**Reply by lrq3000** [reply]
Still using FUTO and still working amazingly well!

**Reply by vickylahkarbytes** [reply]
Yes, but the FUTO English-74 model is slow while transcribing. But on the other hand the CHAT GPT app (android) is very fast. I wonder why this lag in FUTO? Accuracy is same for both the apps.

**Reply by joihn** [reply]
gpt app process audio on a remote powerfull GPU.
FUTO does everything locally.

**Reply by limdingwen** [reply]
Does anyone know if there's a fork of FUTO, or a similar app, that uses the Whisper API?

**Reply by HaiGe120** [reply]
> Does anyone know if there's a fork of FUTO, or a similar app, that uses the Whisper API?

I'm looking into those kind of stuff too, please let me know if you found another. 
You can search in GitHub by the name of 'whisper to input' on Android. you got little issues because it will automatically triggering the full screen input which is really annoying
